sdcm/cluster.py

```diff
@@ -2055,7 +2055,9 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,
         shortid = str(cluster_uuid)[:8]
         name = '%s-%s' % (cluster_prefix, shortid)
         user_data = ('--clustername %s '
-                     '--totalnodes %s' % (name, n_nodes))
+                     '--totalnodes %s '
+                     '--reflector http://reflector.scylladb.com/reflector.php'
```

Comment:
adding the reflector is  not required it will be the default - I prefer that we do not add this as users should not as well.